tests/foreman/ui/test_activationkey.py

```diff
@@ -1209,7 +1209,7 @@ def test_positive_ak_with_custom_product_on_rhel6(session, rhel6_contenthost, ta
     :CaseLevel: Integration
     """
     org = target_sat.api.Organization().create()
-    entities_ids = target_sat.cli_factory.setup_org_for_a_custom_repo(
+    entities_ids = setup_org_for_a_custom_repo(
```

Comment:
Previous was right !

Reply:
Some wrong commit happened. I will correct all this.